spec/jobs/file_inventory_cleanup_job_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require "rails_helper"
 
-RSpec.describe FileInventoryCleanupJob, connect_to_mediaflux: true, type: :job do
+RSpec.describe FileInventoryCleanupJob, connect_to_mediaflux: true, type: :job, integration: true do
```

Comment:
I marked a bunch of tests as integration tests because they rely on projects created in Mediaflux